code/game/objects/effects/spawners/f13lootdrop.dm

```diff
@@ -425,7 +425,7 @@
 	name = "tier 5 armor"
 	loot = list(
 				/obj/effect/spawner/bundle/f13/armor/t45d,
-				/obj/effect/spawner/bundle/f13/armor/t51b				
+				/obj/effect/spawner/bundle/f13/armor/t51b
```

Comment:
```suggestion
				/obj/effect/spawner/bundle/f13/armor/t51b,
```
Make use of trailing commas.
Also if you could remove the extra tabs, that'd be good.

Reply:
Out of scope of this PR, but okay.